log/sequencer_test.go

```diff
@@ -281,12 +281,49 @@ func TestSequenceWithNothingQueued(t *testing.T) {
 	c, ctx := createTestContext(ctrl, params)
 
 	leaves, err := c.sequencer.SequenceBatch(ctx, params.logID, 1)
-	if leaves != 0 {
-		t.Fatalf("Unexpectedly sequenced %d leaves on error", leaves)
+	if leaves != 0 || err != nil {
+		t.Errorf("SequenceBatch()=(%d,%v); want (0,nil)", leaves, err)
 	}
+}
 
+func TestSequenceWithNothingQueuedForceNewRoot(t *testing.T) {
```

Comment:
Done; to preserve consistency in the file I converted everything to be table-driven.

However, I do think that this level of change is out of scope for this pull request.